extensions/positron-r/resources/walkthroughs/workspaces.md

```diff
@@ -0,0 +1,9 @@
+In RStudio, you organize your work using **projects**. Positron doesn't have projects per se, but the concept in Positron that is most analogous to an RStudio project is a **workspace**. You can read more in the VS Code documentation about [what exactly a workspace is](https://code.visualstudio.com/docs/editor/workspaces), but in general think of a workspace as about the same thing as a folder, which is about the same thing as a project in RStudio.
```

Comment:
When I talk about RStudio Projects, I try to always use capital "P", to signal I really mean the very RStudio-specific thing with a `.Rproj` file.

I like this phrasing very much:

> think of a workspace as about the same thing as a folder, which is about the same thing as a project in RStudio.